Formula/python@3.9.rb

```diff
@@ -161,6 +163,7 @@ def install
       f.gsub! "DEFAULT_FRAMEWORK_FALLBACK = [", "DEFAULT_FRAMEWORK_FALLBACK = [ '#{HOMEBREW_PREFIX}/Frameworks',"
     end
 
+    ENV["CPATH"] = cpath.join(":") unless cpath.empty?
```

Comment:
Something like this might be better:
```suggestion
    cpath.each { |path| ENV.append_path "CPATH", path }
```